File: app/src/main/java/com/wxm/camerajob/data/define/EJobStatus.java
package com.wxm.camerajob.data.define;

/**
 * status for camera job
 * Created by ookoo on 2018/2/19.
 */
public enum EJobStatus {
    RUN("running"),
    PAUSE("pause"),
    STOP("stop"),
    UNKNOWN("unknown");

    private String szStatus;

    EJobStatus(String sz)    {
        szStatus = sz;
    }

    /**
     * get status string
     * @return  for status
     */
    public String getStatus()   {
        return szStatus;
    }
}

File: app/src/main/java/com/wxm/camerajob/data/define/EJobStatus.kt
package com.wxm.camerajob.data.define


/**
 * status for camera job
 * Created by WangXM on 2018/2/19.
 */
enum class EJobStatus(val status: String) {
    RUN("running"),
    PAUSE("pause"),
    STOP("stop"),
    UNKNOWN("unknown");
}
